sysvinit/root/etc/init.d/hg-agent

```diff
@@ -1,68 +0,0 @@
-#! /bin/bash
```

Comment:
May I ask you to explain the reason for removing this entire file?

Reply:
Removed support for:
CentOS6, Ubuntu12(Precise Pangolin), Debian7(Wheezy)

sysvinit was used on some old OS versions. We do not need it more.

Reply:
Oh, I see. Okay. Thanks